src/shared/modules/dbMeta/dbMetaEpics.ts

```diff
@@ -437,6 +416,41 @@ export const serverConfigEpic = (some$: any, store: any) =>
     .do(() => store.dispatch(update({ serverConfigDone: true })))
     .mapTo({ type: 'SERVER_CONFIG_DONE' })
 
+export const cleanupSettings = (rawSettings: any) => {
+  const settings: ClientSettings = {
+    allowOutgoingConnections: !isConfigValFalsy(
+      rawSettings['browser.allow_outgoing_connections']
+    ), // default true
+    credentialTimeout: rawSettings['browser.credential_timeout'] || 0,
+    postConnectCmd: rawSettings['browser.post_connect_cmd'] || '',
+    remoteContentHostnameAllowlist:
+      rawSettings['browser.remote_content_hostname_whitelist'] ||
+      initialClientSettings.remoteContentHostnameAllowlist,
+    retainConnectionCredentials: !isConfigValFalsy(
+      rawSettings['browser.retain_connection_credentials']
+    ), // default true
+    retainEditorHistory: !isConfigValFalsy(
+      rawSettings['browser.retain_editor_history']
+    ), // default true
+    // Info: clients.allow_telemetry in versions < 5.0, client.allow_telemetry in versions >= 5.0
+    allowTelemetry: !(
+      isConfigValFalsy(rawSettings['clients.allow_telemetry']) ||
+      isConfigValFalsy(rawSettings['client.allow_telemetry'])
+    ), // default true
+    authEnabled: !isConfigValFalsy(rawSettings['dbms.security.auth_enabled']), // default true
+    // Info: metrics... in versions < 5.0, server.metrics... in versions >= 5.0
+    metricsNamespacesEnabled:
+      isConfigValTruthy(rawSettings['metrics.namespaces.enabled']) ||
+      isConfigValTruthy(rawSettings['server.metrics.namespaces.enabled']), // default false
+    metricsPrefix:
+      rawSettings['metrics.prefix'] ||
```

Comment:
In 5.0 this would just be "undefined" i presume?

Reply:
yes it is undefined